### PR TITLE
Accept localhost for wmts_url in dev

### PR DESCRIPTION
**Pull request by nadine-piveteau**
Related to issue #4070 .
[Testlink](https://mf-geoadmin3.dev.bgdi.ch/ltpin/src/?wmts_url=%2F%2Flocalhost:9020&lang=de&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bav.haltestellen-oev,ch.swisstopo.swisstlm3d-wanderwege&layers_visibility=false,false,false,false&layers_timestamp=18641231,,,&E=2726717.40&N=1078333.54&zoom=4)
No testlink on int given that localhost should only work in dev. 

However, I still get an error 502 Bad Gateway but as far as I understood it's related to the backend and not to my changes (pls correct me if I'm wrong). Any ideas how to tackle this issue? @loicgasser @oterral

### CODE REVIEW

File: src/index.mako.html [hunk 1]
```diff
@@ -742,6 +742,9 @@
         var proxyUrl = prtl + '${proxy_url}' + '/';
         var apiOverwrite = !!(getParam('api_url') || env);
         var wmtsUrl = setBackend(prtl + '${wmts_url}', prtl + '${wmts_tech_url}', 'wmts_url');
+        if (wmtsUrl.includes('localhost') ){
```
Comment (oterral) [inline src/index.mako.html]: Use a regex , `includes` is not supported by IE11:
/localhost/.test( wmtsUrl)

File: rc_dev [hunk 2]
```diff
@@ -12,3 +12,4 @@ export PUBLIC_URL=//public.dev.bgdi.ch
 export PRINT_URL=//service-print.dev.bgdi.ch
 export PROXY_URL=//service-proxy.dev.bgdi.ch
 export WMTS_URL=//tod.dev.bgdi.ch
+export ADMIN_URL_REGEXP="^(ftp|http|https):\/\/(.*(\.bgdi|\.geo\.admin)\.ch)|(https?:\/\/localhost:[0-9]{1,9})"
```
Comment (oterral) [inline rc_dev]: You can simplify a bit:
```
"^(ftp|http|https):\/\/(.*(\.bgdi|\.geo\.admin)\.ch|localhost:[0-9]{1,9})"
```

Comment (oterral) [inline rc_dev]: Is the maximal number of port contains 9 numbers?

Comment (nadine-piveteau) [reply rc_dev]: The port number is an unsigned 16-bit integer which means that the maximum port number is 2^16-1=65535. I changed it in the last commit.

**Comment by oterral** [comment]
@ltclm can you test ?

**Comment by ltclm** [comment]
everything is working fine, thank you!

**Pull request merged**